### PR TITLE
feat(completions/btrfs): fix no completion after the pipeline

### PR DESCRIPTION
**Pull request by akiirui**
## Description

Before: `btrfs command | btrfs <TAB>` no completion
After : `btrfs command | btrfs <TAB>` have completion

use `not __fish_seen_subcommand_from` instead of `__fish_is_first_arg`

Fixes issue #

## TODOs:

- [ ] Changes to fish usage are reflected in user documentation/manpages.
- [ ] Tests have been added for regressions fixed
- [ ] User-visible changes noted in CHANGELOG.rst

### CODE REVIEW

File: share/completions/btrfs.fish [hunk 1]
```diff
@@ -56,373 +58,373 @@ set -l replace '__btrfs_command_groups replace'
...
 # btrfs inspect-internal logical-resolve
-complete -f -c btrfs -n '__btrfs_command_groups inspect-internal logical-resolve' -s P -d 'Skip the path resolving and print the inodes instead'
-complete -f -c btrfs -n '__btrfs_command_groups inspect-internal logical-resolve' -s v -d 'Verbose mode'
-complete -f -c btrfs -n '__btrfs_command_groups inspect-internal logical-resolve' -s s -d 'Set inode container\'s SIZE'
+complete -f -c btrfs -n "$inspect-internal logical-resolve" -s P -d 'Skip the path resolving and print the inodes instead'
+complete -f -c btrfs -n "$inspect-internal logical-resolve" -s v -d 'Verbose mode'
+complete -f -c btrfs -n "$inspect-internal logical-resolve" -s s -d 'Set inode container\'s SIZE'
 # btrfs inspect-internal min-dev-size
-complete -f -c btrfs -n '__btrfs_command_groups inspect-internal min-dev-size' -l id -d 'Specify the DEVICE-ID to query'
+complete -f -c btrfs -n "$inspect-internal min-dev-size" -l id -d 'Specify the DEVICE-ID to query'
```
Comment (krobelus) [inline share/completions/btrfs.fish:349]: These should be `$inspect_internal` (`_` instead of `-`)

File: share/completions/btrfs.fish [hunk 2]
```diff
@@ -56,373 +58,373 @@ set -l replace '__btrfs_command_groups replace'
...
-complete -f -c btrfs -n $check -l force -d 'Skip mount checks, repair is not possible'
-complete -f -c btrfs -n $check -l mode -d 'Allows choice of memory/IO trade-offs' -ra '{original,lowmem}'
-complete -f -c btrfs -n $check -l init-csum-tree -d 'Create a new CRC tree (repair only)'
-complete -f -c btrfs -n $check -l init-extent-tree -d 'Create a new extent tree (repair only)'
-complete -f -c btrfs -n $check -l clear-space-cache -d 'clear space cache (repair only)' -ra '{v1,v2}'
-complete -f -c btrfs -n $check -l check-data-csum -d 'Verify checksums of data blocks'
-complete -f -c btrfs -n $check -s Q -l qgroup-report -d 'Print a report on qgroup consistency'
-complete -f -c btrfs -n $check -s E -l subvol-extents -d 'Print subvolume extents and sharing state'
-complete -f -c btrfs -n $check -s p -l progress -d 'Indicate progress'
+complete -f -c btrfs -n "$check" -s s -l super -d 'Use this SUPERBLOCK copy'
```
Comment (krobelus) [inline share/completions/btrfs.fish:83]: I guess we don't need to add quotes

**Review by krobelus — COMMENTED** [review]
Thanks. I think I managed to fix the problem with the pipe in a more general way in 701b7450d.

**Comment by akiirui** [comment]
> Thanks. I think I managed to fix the problem with the pipe in a more general way in [701b745](https://github.com/fish-shell/fish-shell/commit/701b7450dfeb8bdddf91c433ff696dfe3d996b9b).

Yep, that is better than this. I think this PR can be closed (the second commit isn't important)

**Pull request closed without merging**